react-native/react/native/pinentry.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+/* @flow */
```

Comment:
yep! unless you are typing this out, otherwise you can create needless flow type errors